src/function.rs

```diff
@@ -79,6 +80,15 @@ fn arch(_context: &FunctionContext) -> Result<String, String> {
   Ok(target::arch().to_owned())
 }
 
+fn capitalize(_context: &FunctionContext, s: &str) -> Result<String, String> {
+  let mut chars = s.chars();
+  match chars.next() {
+      None => Ok(String::new()),
+      Some(c) => Ok(c.to_uppercase().collect::<String>() +
+                    &chars.as_str().to_lowercase()),
+  }
```

Comment:
Maybe something like this:

```suggestion
  Ok(s.chars().enumerate().flat_map(|(i, c)| if i == 0 { c.upper_case() } else { c.lower_case() }).collect())
```

Reply:
So much more elegant! Applied with rustfmt.